benchmarks.txt

```diff
@@ -0,0 +1,2990 @@
+flag provided but not defined: -vvv
```

Comment:
I’m guessing this was checked in by mistake? add to gitignore?

Reply:
Yip, this wasn't supposed to be added. Removed it

Reply:
yip